VultisigApp/VultisigApp/Services/Blowfish/BlowfishService.swift

```diff
@@ -18,7 +18,7 @@ struct BlowfishService {
         origin: String,
         txObjects: [BlowfishRequest.BlowfishTxObject],
         simulatorConfig: BlowfishRequest.BlowfishSimulatorConfig? = nil
-    ) async throws -> BlowfishResponse? {
+    ) async -> BlowfishResponse? {
```

Comment:
throws was fine here, let's not return optional instead?

Reply:
`do` `catch`  should be on the top level only. eg in viewModel

Reply:
Blowfish is a 3rd party API; we have no control over it. I prefer to return it as nil since it was frequently breaking, so in that case, for Blowfish, it is better to only show something to break our transaction flow.